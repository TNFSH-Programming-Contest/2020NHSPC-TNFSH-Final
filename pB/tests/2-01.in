85
-9.576 246.468
55.768 241.441
45.715 173.584
144.988 51.692
161.324 233.901
200.28 208.769
229.0 237.671
332.225 254.007
286.986 213.796
224.155 163.531
260.597 148.451
343.534 199.973
428.984 203.743
435.267 98.187
401.339 88.134
397.569 153.478
339.764 153.478
339.764 109.496
185.2 117.036
181.43 0.17
111.06 -8.626
121.113 -29.989
177.66 -36.272
271.907 -114.182
308.349 -213.455
357.357 -239.844
335.994 -139.314
425.214 -146.854
397.569 -92.82
325.941 -36.272
332.225 66.771
474.223 69.284
471.709 232.645
532.027 237.671
586.062 158.504
557.159 -66.43
525.744 26.559
381.233 20.276
390.029 -36.272
490.559 -27.475
538.31 -413.257
483.019 -413.257
484.276 -188.323
381.233 -188.323
408.878 -285.082
398.825 -419.541
270.65 -454.726
186.457 -450.956
195.253 -405.718
219.129 -443.416
289.5 -405.718
307.092 -288.852
258.084 -194.606
201.536 -107.899
186.457 -156.907
241.748 -251.154
241.748 -404.461
168.864 -341.63
136.192 -410.744
98.493 -321.524
59.538 -380.585
28.123 -344.143
6.4 -321.6
-53.558 -326.551
-73.663 -270.003
-17.116 -253.667
40.689 -313.984
171.377 -249.897
107.29 -161.934
57.025 -177.013
-18.372 -207.172
-81.203 -122.978
-43.505 -55.121
25.609 -99.0
111.06 -81.51
41.946 78.081
-5.806 83.107
10.53 -38.785
-89.999 25.303
-89.999 96.93
-49.788 61.745
-37.221 133.372
-101.309 163.531
-91.256 231.388
1.734 181.123
59.538 -380.585
