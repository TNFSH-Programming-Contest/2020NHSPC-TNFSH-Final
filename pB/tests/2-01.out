1902191.427